### PR TITLE
Expose (and use) syntax tree outside the parser

### PR DESCRIPTION
**Pull request by kriskowal**
r @abhinav

### CODE REVIEW

**Comment by abhinav** [comment]
This appears to have code that has already been reviewed/merged. You probably need to rebase.

File: thrift-idl.js [hunk 1]
```diff
@@ -20,15 +20,20 @@
...
+
 var PEG = require('pegjs');
 var fs = require('fs');
 var path = require('path');
+var ast = require('./ast');
 
 var grammarPath = path.join(__dirname, 'thrift-idl.pegjs');
-var grammar = PEG.buildParser(fs.readFileSync(grammarPath).toString('ascii'));
+var parserSource = PEG.buildParser(fs.readFileSync(grammarPath).toString('ascii'), {output: 'source'});
+var parserMaker = new Function('ast', 'return ' + parserSource);
```
Comment (kriskowal) [inline thrift-idl.js]: these are not the droids you are looking for

Comment (abhinav) [inline thrift-idl.js]: ew

**Comment by abhinav** [comment]
:ok:

**Pull request merged**